packages/tss-client/src/client.ts

```diff
@@ -43,9 +43,13 @@ if (globalThis.js_read_msg === undefined) {
             clearInterval(timer);
             resolve(found.msg_data);
           }
-          if (counter >= 500) {
+          if (counter >= 3000) {
```

Comment:
The servers will only wait for the client to respond for 5 seconds at max, this only updates it on one end. Which mean you're proposing holding onto a worker for a very long time in the worst case...

Reply:
Again, you are not correct in what you are saying here.

The read timeout at the server side is currently at 50 seconds.
[dklsPrecomputeWorker.ts#L66](https://github.com/torusresearch/tss-server/blob/21555be75409466d311da0c3d28a01f654d491e3/packages/server/src/app/v1/tss-dkls/pool/dklsPrecomputeWorker.ts#L66)

I've tested that increasing this read timeout here increases the stability of the client on a slow connection. You can test yourself by using Chrome's throttling mechanism and setting it to "slow 3G". The signing test then fails consistently with the current read timeout of 5 seconds, but succeeds consistently if the read timeout is increased like above.

Reply:
Ok, you found a typo that got past a review somewhere, it should be (500,10) and not (5000,10).

Please don't link public repositories to private ones in future though. :)